Update sdk-deprecation.mdx (#1025)
ethereum-optimism/docs: pages/builders/notices/sdk-deprecation.mdx

For further assistance or questions about this migration, feel free to reach through the following channels:

*   Connect with us on [Discord](https://discord.gg/optimism) for community support

> *   Ask a question on our [developer forum](https://github.com/ethereum-optimism/developers/discussions) for discussions and support

Reviewer: ```suggestion
*   Join us on our [developer forum](https://github.com/ethereum-optimism/developers/discussions) for discussions, questions, and general support.
```
